source/agora/node/Ledger.d

```diff
@@ -18,8 +18,10 @@ import agora.common.Block;
 import agora.common.Data;
 import agora.common.Hash;
 import agora.common.Transaction;
+import agora.node.LedgerOutputStream;
 
 import std.algorithm;
+import std.mmfile;
```

Comment:
Superfluous?

Reply:
It's a necessary part of the next version, but I'll delete it now.